lib/PerlIO.pm

```diff
@@ -235,12 +235,6 @@ like C<:utf8>.  An example of a possible use might be:
 
 A more elegant (and safer) interface is needed.
 
-=item :win32
```

Comment:
I wonder if it would be useful to retain this documentation for historical reasons but mention the version it was removed in. (Don't see any reason to retain any of the other docs)

Reply:
It might have been if anyone had actually used it, but they didn't.